src/extension.ts

```diff
@@ -69,6 +77,25 @@ export function activate (context: vscode.ExtensionContext) {
   commandManager.register(new commands.SaveHTML(engine))
   commandManager.register(new commands.SaveDocbook(engine))
 
+  const antora = new AntoraSupport()
+  if (await antora.isDetected()) {
+    const config = vscode.workspace.getConfiguration('asciidoc', null)
+    if (config.get('antora') === false && config.get('antoraConfigIsSet') === false) {
```

Comment:
We should use https://code.visualstudio.com/api/references/vscode-api#WorkspaceConfiguration to check that `isEnableAntoraSupportSettingDefined` is `undefined`.